DevArt.Users.Application/Dto/UpdateUserDto.cs

```diff
@@ -5,6 +5,6 @@ public class UpdateUserDto
     public string? NewPassword { get; set; } = string.Empty;
 
     public string? PasswordConfirmation { get; set; } = string.Empty;
-    
-    public string? NickName { get; set; }
+
+    public string? NickName { get; set; } = string.Empty;
```

Comment:
You've ready gave it an initializer, so it should never be nullable?
```suggestion
    public string NickName { get; set; } = string.Empty;
```